### PR TITLE
Add Evaluation guide

### PR DESCRIPTION
**Pull request by meinradandermatt**
# Motivation
We would like to have an entry point for people they start with the evaluation from livingdocs. It should be a help to start with the setup and know which step they have to do and read through the documentation.

### CODE REVIEW

File: walkthroughs/evaluation_guide.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Evaluation guide
+
+## Short introduction to livingdocs
+Livingdocs has 2 main applications, the livingdocs-editor and the livingdocs-server. For the livingdocs-server we need the following additional application:
```
Comment (gabrielhase) [inline walkthroughs/evaluation_guide.md]: `application` -> `applications`

File: walkthroughs/evaluation_guide.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- elasticsearch
+- postgres
+- redis
+- Amazon s3 buckets
+  - Image bucket
+  - Design bucket
+- Image service (Imgix)
+- Pusher
```
Comment (gabrielhase) [inline walkthroughs/evaluation_guide.md]: Amazon Mail server?

Comment (gabrielhase) [reply walkthroughs/evaluation_guide.md]: IFramely?

File: walkthroughs/evaluation_guide.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+At the moment you have the blank design without any assets on your design server which has his files on the S3 bucket. The next step is now to create your own design. 
+
+[create design](../reference-docs/common-designs/create_designs)
+
+## Metadata
+The next step is to create your own metadata field with a own metadata plugin. 
+
+[create metadata field](../reference-docs/common-designs/create_designs)
+
+## doc-include
```
Comment (gabrielhase) [inline walkthroughs/evaluation_guide.md]: I would say that things like delivery (webhooks) and import come first for a customer

File: walkthroughs/evaluation_guide.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Evaluation guide
 
 ## Short introduction to livingdocs
-Livingdocs has 2 main applications, the livingdocs-editor and the livingdocs-server. For the livingdocs-server we need the following additional application:
+Livingdocs has 2 main application, the livingdocs-editor and the livingdocs-server. For the livingdocs-server we need the following additional application:
```
Comment (peyerluk) [inline walkthroughs/evaluation_guide.md]: Wasn't it better before ;)

File: walkthroughs/evaluation_guide.md [hunk 5]
```diff
@@ -1,25 +1,33 @@
...
-  - Design bucket
-- Image service (Imgix)
-- Pusher
-- Amazon Mailserver
-- Iframely
-
-Elasticsearch, Postgres and redis are started in a docker container. The other 3 are external services. For evaluation we already have added accounts for this 3 services in the config files. They are limited to a specific amount of data but this shouldn't be a problem for the evaluation.
+Livingdocs has 2 main application, the livingdocs-editor and the livingdocs-server. For the livingdocs-server we need the following additional application.
+
+#### livingdocs-server dependencies
```
Comment (peyerluk) [inline walkthroughs/evaluation_guide.md]: After you changes  this title is somewhat outdated.

maybe:
```
#### Livingdocs Components, Containers and external Services
```

File: walkthroughs/evaluation_guide.md (hunk 3 above)
Comment (meinradandermatt) [reply walkthroughs/evaluation_guide.md]: I added this two first

**Pull request merged**